Pod/Classes/Categories/UIApplication+PreferredContentSizeCategoryMock.h

```diff
@@ -8,8 +8,16 @@
 
 @import UIKit;
 
+/**
+ `UIApplication` extension for `-preferredContentSizeCategory` property mocking.
+ */
 @interface UIApplication (PreferredContentSizeCategoryMock)
 
+
+/**
+ Sets `UIContentSizeCategory` value to mock.
+ To disable mocking `-preferredContentSizeCategory`, set property to `nil`.
```

Comment:
Maybe it will be more clear if you had explicit method called `stopMocking`?

Reply:
`AGPreferredContentSizeCategoryMocker` actually do this.

https://github.com/Grubas7/AGSnapshotHelper/blob/master/Pod/Classes/AGPreferredContentSizeCategoryMocker.h

Reply:
So do we need that here?

Reply:
`UIApplication+PreferredContentSizeCategoryMock` have only swizzling logic (replacement for OCMock), and I think it's nice to have this in separated, reusable class.

Beside I would like to avoid explicit calling:
```
UIApplication.shared.mockedContentSizeCategory = .small
...
UIApplication.shared.mockedContentSizeCategory = nil
```

IMO more clearly is:
```
let applicationMock = PreferredContentSizeCategoryMocker()
applicationMock.startMockingContentSizeCategory(.small)
...
applicationMock.stopMocking()
```

Reply:
What I actually meant, is that the second option (with `stopMocking`), is better. 🙂 I think we misunderstood each other.

If we both agreed that `stopMocking` is better, why the library is supporting setting `mockedContentSizeCategory` to `nil` in order to stop mocking?

Reply:
Done in PR no. 15.